notebooks/MIRI_LRS_spectral_extraction/miri_lrs_advanced_extraction_part1.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Hi @orifox, I suggest you update the code as follows for a more concise code: </p><p> </p><pre class="ql-syntax" spellcheck="false">data_tar_url = 'https://data.science.stsci.edu/redirect/JWST/jwst-data_analysis_tools/MIRI_LRS_notebook/data.tar.gz'
# Download and unpack data if needed
if not os.path.exists("data.tar.gz"):
&nbsp; &nbsp; print("Downloading Data")
&nbsp; &nbsp; urllib.request.urlretrieve(data_tar_url, 'data.tar.gz')


if not os.path.exists("data/"):
&nbsp; &nbsp; print("Unpacking Data")
&nbsp; &nbsp; with tarfile.open('./data.tar.gz', "r:gz") as tar:
&nbsp; &nbsp; &nbsp; &nbsp; tar.extractall()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>